src/data/markdown/docs/30 k6-browser/02 API/08 Locator/textcontent--options--.md

```diff
@@ -24,21 +24,23 @@ Returns the `element.textContent`.
 
 <CodeGroup labels={[]}>
 
-<!-- eslint-skip -->
-
 ```javascript
-page
-  .goto('https://test.k6.io/browser.php')
-  .then(() => {
-    const options = page.locator("#checkbox1");
-    console.log(options.textContent()); /*  Zero
-                                            One
-                                            Two
-                                            Three
-                                            Four
-                                            Five
-                                        */    
-  });
+import { chromium } from 'k6/experimental/browser';
+
+export default async function () {
+  const browser = chromium.launch();
+  const page = browser.newPage();
+  
+  await page.goto('https://test.k6.io/browser.php');
+	const options = page.locator("#checkbox1");
+	console.log(options.textContent()); /*  Zero
+																					One
+																					Two
+																					Three
+																					Four
+																					Five
+																			*/    
```

Comment:
Did these counting numbers move from previous version? They do seem a little bit too much to the right on my screen.

Reply:
Hmm this shouldn't be included. Let me change